### ISSUE TITLE
[Eclipse-ThirdParty][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Inappropriate control type for the Table in "Install Details" dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1377]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Help menu.
3. Navigate and select Install new software Menu item under Help menu.
4. Install dialog will open.
5. In "Install" dialog , within the Work with text box, type http://dl.microsoft.com/eclipse/.
6. Navigate to and check Azure Toolkit for Eclipse, and uncheck Contact all update sites during install to find required software.
7. Select "Next" button.
8. Under "Install Details" heading ,  tab to table and observe the control type using inspect.

**Actual Result:**
Control type for table is pane.

**Narrator :**
Narrator announces as "Not on table" while using the table commands such as "Caps+F3", "Caps+F8". Also in scan mode, on using "Ctrl+alt+arrow" keys to navigate the table cells, narrator just announces as "Not on table".

**Expected Result:**
Control type should be Table.
Narrator table commands such as "Caps+F3", "Caps+F8" should work.

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![install details-table-not defined](https://user-images.githubusercontent.com/22956130/35681163-91b8667c-0782-11e8-919a-7289cfd1e1ca.jpg)

**Comment by Eskibear** [comment]
duplicated with #1405

**Comment by Eskibear** [comment]
Moreover, it's the Eclipse IDE's dialog, out of the toolkit's scope

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749812
